tutorials/get-started-kubernetes-loadbalancer/index.mdx

```diff
@@ -77,106 +77,81 @@ It is not possible to create the external Scaleway Load Balancer yourself via th
 ## Step 1: Create Load Balancer from YAML manifest
 
 1. Open a terminal, and run the following command to check the state of your cluster:
-
-  ```
-  kubectl get all
-  ```
-
-  You should see two pods, the NodePport Service created during the previous tutorial (along with the basic default ClusterIP Service), the deployment and the two ReplicaSets.
-
+    ```
+    kubectl get all
+    ```
+    You should see two pods, the NodePport Service created during the previous tutorial (along with the basic default ClusterIP Service), the deployment and the two ReplicaSets.
 2. Delete the NodePort Service with the following command:
-
-  ```
-  kubectl delete svc name-of-nodeport-service
-  ```
- 
-  In our case, we called our NodePort service `mydeployment`, so the command is `kubectl delete svc mydeployment`.
-
+    ```
+    kubectl delete svc name-of-nodeport-service
+    ```
+    In our case, we called our NodePort service `mydeployment`, so the command is `kubectl delete svc mydeployment`.
 3. Create a new file called `lb.yaml`:
-
-  ```
-  nano lb.yaml
-  ```
-
-  This file will be a manifest for our LoadBalancer Service.
-
+    ```
+    nano lb.yaml
+    ```
+    This file will be a manifest for our LoadBalancer Service.
 4. Copy and paste the following text into the file:
-
-  ```
-  apiVersion: v1
-  kind: Service
-  metadata:
-    name: myloadbalancer
-    labels:
-      app: mydeployment
-  spec:
-    type: LoadBalancer
-    ports:
-    - port: 8000
-      name: http
-      targetPort: 8000
-    selector:
-      app: mydeployment
-  ```
-
+    ```
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: myloadbalancer
+      labels:
+        app: mydeployment
+    spec:
+      type: LoadBalancer
+      ports:
+      - port: 8000
+        name: http
+        targetPort: 8000
+      selector:
+        app: mydeployment
+    ```
     - **apiVersion** specifies which verison of the Kubernetes API to use to create the object
     - **kind** specifies the kind of object defined in this yaml file, here a Service
     - **metadata** helps uniquely identify our Service object: we give it a name (`myloadbalancer`), and a label.
     - **spec** specifies the Service. It is of **type** LoadBalancer. We then go on to specify its **ports**. We can define many ports if we want but here we just specify the necessary port `8000` for our `whoami` app. Since `8000` is an HTTP port, we add a name tag and call it `http`. **targetPort** is the port the container accepts traffic (in our case necessarily `8000`), **port** is the abstracted Service port. For simplicity, we set both as `8000`, though we could change **port** to something else.
```

Comment:
```suggestion
    - **spec** specifies the Service. It is of LoadBalancer **type**. We then go on to specify its **ports**. We can define many ports if we want but here we just specify the necessary port `8000` for our `whoami` app. Since `8000` is an HTTP port, we add a name tag and call it `http`. **targetPort** is the port the container welcomes traffic (in our case necessarily `8000`), **port** is the abstracted Service port. For simplicity, we set both as `8000`, though we could change **port** to something else.
```